students/jakielczyk_miroslaw/lesson_03_functions/Collatz_Sequence.py

```diff
@@ -0,0 +1,16 @@
+# Collatz Sequence
+print ("Enter the number ")
+number = int(input())
+def collatz(number):
+    if number % 2 == 0:
+        print (number // 2)
+        return (number // 2)
+    elif number % 2 == 1:
+        print (3 * number + 1)
```

Comment:
it would be better to have single `print` outside of if construct.
Or even better to move it to while loop and use `print(result)`